src/library-aux/scala/Null.scala

```diff
@@ -13,9 +13,13 @@
 package scala
 
 /** `Null` is - together with [[scala.Nothing]] - at the bottom of the Scala type hierarchy.
- *
- *  `Null` is a subtype of all reference types; its only instance is the `null` reference.
- *  Since `Null` is not a subtype of value types, `null` is not a member of any such type.  For instance,
- *  it is not possible to assign `null` to a variable of type [[scala.Int]].
- */
+  *
+  * `Null` is the type of the `null` literal. It is a subtype of every type
+  * except those of value classes (that is, classes which are subtypes of [[AnyVal]]).
```

Comment:
Still kind of confusing to refer back to the "except ..." clause with "this includes", so: "the exception includes"? Does that contract to "exclusion"?

Suggestion line 1:

```suggestion
  * except those of value classes. Value classes are subclasses of [[AnyVal]], which includes
```